### PR TITLE
Pull-Request: added "join table" for visits

### PR DESCRIPTION
**Pull request by Zven94**
### **Vet Clinic**

In this project, added some many-to-many relationships and wrote more complex queries to use them to answer questions.

**Queries**

_Who was the last animal seen by William Tatcher?_
last_animal_william

_How many different animals did Stephanie Mendez see?_
how_many_stephanie

_List all vets and their specialties, including vets with no specialties_
vets_and_specializations

_List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020_
visits_stephanie_between_apr_aug

_What animal has the most visits to vets?_
most_visit

_Who was Maisy Smith's first visit?_
first_visit_maisy

_Details for most recent visit: animal information, vet information, and date of visit_
last_visit

_How many visits were with a vet that did not specialize in that animal's species?_
visits_no_specializations

_What specialty should Maisy Smith consider getting? Look for the species she gets the most_
specialty_for_Maisy

### CODE REVIEW

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Zven94 ✋

Good job so far!

## Highlights!
- Added screenshots to the pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] Kindly ensure your `schema.sql` file contains code for creating the tables of `vets`, `specialization`, and `visits`.
- [x] Kindly ensure your `data.sql` file contains all the data added to the `vets`, `specialization`, and `visits` tables.
- [x] Kindly ensure your `queries.sql` file contains code for all the queries to answer the following questions in transactions.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql [hunk 1]
```diff
@@ -63,3 +63,45 @@ vet_clinic=# UPDATE animals AS a SET owner_id = o.id FROM owners AS o WHERE a.na
...
+vet_clinic-# ('Jack Harkness',38,'2008/06/08');
+
+/*  INSERT DATA TO SPECIALIZATIONS TABLE   */ 
+
+vet_clinic=# INSERT INTO specializations (id,idvets,idspecies)
+vet_clinic-# VALUES (1,1),
+vet_clinic-# (3,1),
+vet_clinic-# (3,2),
+vet_clinic-# (4,2);
+
```
Comment (Owusu-Desmond) [inline data.sql]: - [x] I suggest removing the unnecessary command line `vet_clinic-#` from data.sql to avoid errors when executing the file. :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Zven94 , :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Zven94 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done deleting the commented code. ✔
- The milestone requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**